server/views/register.hbs

```diff
@@ -8,7 +8,7 @@
     <label>Name</label>
     <input type="text" class="form-control" placeholder="Name" name="name">
   </div>
-  <div class="form-group">
+  <div class="form-group"></div>
```

Comment:
why did you change this?

Reply:
Fix this part. It was me wrong. and add it unintentionally